storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst

```diff
@@ -8,7 +8,18 @@ The purpose of |xbcloud| is to download from the cloud and upload to the cloud t
 invoked as a pipeline with |xtrabackup| to stream directly to the cloud without
 needing a local storage.
 
-|xbcloud| stores each chunk as a separate object with a name
+.. note::
+
+   In a Bash shell, the ``$?`` parameter returns the exit code from the last binary. If you use pipes the ``${PIPESTATUS[x]}`` array parameter returns the exit codes for each binary in the pipe string. 
+
+   .. code-block::
+
+      $ xtrabackup --backup --stream=xbstream --target-dir=/storage/backups/ | xbcloud put [options] full_backup
```

Comment:
Same as 2.4 review, use a simple `true | false` to show how it works

Reply:
Done.